### ISSUE TITLE
Add ts-ify pipeline into build command

### ISSUE THREAD

**Issue by dghelm** [SkynetLabs/skynet-js#75]
We should auto-build the browser version for script-tag imports on build, or give it an additional build command.

We'd need to add browserify, tsify and uglifyjs to dev dependencies. Need to look into package.json usage or config files, but the browser commandline is:

`browserify ./src/index.ts -s skynet -p [ tsify --noImplicitAny ] > ./dist/browser/index.js`
`uglifyjs ./dist/browser/index.js -o ./dist/browser/index.min.js -c -m`

**Comment by m-cat** [comment]
Note that the `webpack` branch already builds bundles for script-tag imports. I'm not sure what the pros/cons of webpack vs. browserify are. At this point, much of the `webpack` branch may be obsolete anyway.

**Comment by dghelm** [comment]
True, I'm not sure what the merge will look like, but Parox also sent this over, which may be a nice direction to move: https://tsdx.io/

I think this is probably a farther-off to-do that we can look at with nodejs (and command-line!) additions to the SDK.
